splinter

```diff
@@ -46,11 +46,16 @@ main() {
   local i
   for i in "${!regexes[@]}"; do
     local matches
-    matches="$(grep -EHInr "${regexes[$i]}" "${src_paths[@]}" || echo)"
+    local should_ignore
+    matches="$(grep -EHInr "${regexes[$i]}" "$PWD/${src_paths[@]}" || echo)"
+
     if [ -n "$matches" ]; then
-      violations+="$(
-        echo "$matches" | cut -d: -f-2 | sed "s#\$#:${messages[$i]}#g" \
-      )"$'\n'
+      while IFS= read -r match; do
```

Comment:
Simpler to [grep once](https://github.com/duolingo/splinter/commit/afb52564cb935d0baa92895cf46de658e77e475f). I've released that and other improvements (e.g. unconflictable [sed delimiter](https://github.com/duolingo/splinter/commit/bdfde50377bc74a90c8b95ae415b7ed7d04b11de)) as [1.4.0](https://github.com/duolingo/splinter/releases)